gcloud/search/test_index.py

```diff
@@ -105,9 +105,168 @@ def test_from_api_repr_w_properties(self):
         self.assertTrue(index._client is client)
         self._verifyResourceProperties(index, RESOURCE)
 
+    def test_list_documents_defaults(self):
```

Comment:
d8961e5 factors out the common document setup / verification code from the two `list_documents` testcase methods.